### ISSUE TITLE
Tests for version 6.3 fail when run from inside egg/wheel

### ISSUE THREAD

**Issue by icemac** [zopefoundation/zope.testrunner#163]
See https://github.com/zopefoundation/zopetoolkit/actions/runs/7828690859/job/21359091342?pr=74#step:6:572

**Comment by mtelka** [comment]
I see this on OpenIndiana too.

For reference, the failure is this:
```
Failure in test test_xml_report_with_failures (zope.testrunner.tests.test_xml_output.TextXMLOutputWithErrors)
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/unittest/case.py", line 60, in testPartExecutor
    yield
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/unittest/case.py", line 676, in run
    self._callTestMethod(testMethod)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/unittest/case.py", line 633, in _callTestMethod
    method()
  File "/home/runner/work/zopetoolkit/zopetoolkit/.tox/py38/eggs/zope.testrunner-6.3-py3.8.egg/zope/testrunner/tests/test_xml_output.py", line 171, in test_xml_report_with_failures
    self.assertTrue(report.exists())
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/unittest/case.py", line 765, in assertTrue
    raise self.failureException(msg)
AssertionError: False is not true
```

**Comment by gforcada** [comment]
Interestingly from `master` branch, both `tox -e py311` or `tox -e py38` do pass... 🤔

**Comment by mtelka** [comment]
> Interestingly from `master` branch, both `tox -e py311` or `tox -e py38` do pass... 🤔

I assume this is because some file is missing from sdist.

**Comment by icemac** [comment]
When running just the failing test it also fails on `master`. It seems that we have a test isolation problem here.

I am already working on it.

**Comment by icemac** [comment]
Fixed inb 6.3.1 release.
